### ISSUE TITLE
issue installing ,ibm_db v2.6.2 in docker

### ISSUE THREAD

**Issue by huineng** [ibmdb/node-ibm_db#599]
I can no longer install the new ibm_db v2.6.2 in docker, not sure exactly what changed in this new version or what upgrade needs to be done

i'm using FROM node:13.2.0

by the way , i saw a reference on docker but yum is not available in docker base image

if something for this version now needs additional installation, can you provide them in apt-get format

or maybe this is a bug 

as you know i never had any problems in docker

complete log 

```
make: Entering directory '/home/app/node_modules/ibm_db/build'
  CXX(target) Release/obj.target/odbc_bindings/src/odbc.o
odbc_bindings.target.mk:116: recipe for target 'Release/obj.target/odbc_bindings/src/odbc.o' failed
make: Leaving directory '/home/app/node_modules/ibm_db/build'

Error: Command failed: node-gyp configure build  --IS_DOWNLOADED=true --IBM_DB_HOME="$IBM_DB_HOME"
../src/odbc.cpp: In static member function ‘static void ODBC::UV_AfterCreateConnection(uv_work_t*, int)’:
../src/odbc.cpp:215:27: warning: ‘v8::Local<v8::Value> Nan::Callback::Call(int, v8::Local<v8::Value>*) const’ is deprecated [-Wdeprecated-declarations]
     data->cb->Call(1, info);
                           ^
In file included from ../src/odbc.h:23:0,
                 from ../src/odbc.cpp:29:
../../nan/nan.h:1740:3: note: declared here
   Call(int argc, v8::Local<v8::Value> argv[]) const {
   ^~~~
../src/odbc.cpp:227:27: warning: ‘v8::Local<v8::Value> Nan::Callback::Call(int, v8::Local<v8::Value>*) const’ is deprecated [-Wdeprecated-declarations]
     data->cb->Call(2, info);
                           ^
In file included from ../src/odbc.h:23:0,
                 from ../src/odbc.cpp:29:
../../nan/nan.h:1740:3: note: declared here
   Call(int argc, v8::Local<v8::Value> argv[]) const {
   ^~~~
../src/odbc.cpp: In static member function ‘static void ODBC::GetStringParam(v8::Local<v8::Value>, Parameter*, int)’:
../src/odbc.cpp:997:51: error: no matching function for call to ‘v8::Value::ToString(v8::Isolate*)’
     Local<String> string = value->ToString(ISOLATE);
                                                   ^
In file included from ../src/odbc.cpp:23:0:
/home/app/.cache/node-gyp/13.2.0/include/node/v8.h:2750:44: note: candidate: v8::MaybeLocal<v8::String> v8::Value::ToString(v8::Local<v8::Context>) const
   V8_WARN_UNUSED_RESULT MaybeLocal<String> ToString(
                                            ^~~~~~~~
/home/app/.cache/node-gyp/13.2.0/include/node/v8.h:2750:44: note:   no known conversion for argument 1 from ‘v8::Isolate*’ to ‘v8::Local<v8::Context>’
../src/odbc.cpp: In static member function ‘static v8::Local<v8::Value> ODBC::CallbackSQLError(SQLSMALLINT, SQLHANDLE, char*, Nan::Callback*)’:
../src/odbc.cpp:1235:19: warning: ‘v8::Local<v8::Value> Nan::Callback::Call(int, v8::Local<v8::Value>*) const’ is deprecated [-Wdeprecated-declarations]
   cb->Call(1, info);
                   ^
In file included from ../src/odbc.h:23:0,
                 from ../src/odbc.cpp:29:
../../nan/nan.h:1740:3: note: declared here
   Call(int argc, v8::Local<v8::Value> argv[]) const {
   ^~~~
make: *** [Release/obj.target/odbc_bindings/src/odbc.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:270:12)
gyp ERR! System Linux 4.9.184-linuxkit
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build" "--IS_DOWNLOADED=true" "--IBM_DB_HOME=/home/app/node_modules/ibm_db/installer/clidriver"
gyp ERR! cwd /home/app/node_modules/ibm_db
gyp ERR! node -v v13.2.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 

    at ChildProcess.exithandler (child_process.js:295:12)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1023:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:281:5) {
  killed: false,
  code: 1,
  signal: null,
  cmd: 'node-gyp configure build  --IS_DOWNLOADED=true --IBM_DB_HOME="$IBM_DB_HOME"'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ibm_db@2.6.2 install: `node installer/driverInstall.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ibm_db@2.6.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/app/.npm/_logs/2019-12-09T22_58_39_430Z-debug.log
```

**Comment by mitchdennett** [comment]
I’m having the same issue but when installing on macOS 

https://github.com/ibmdb/node-ibm_db/issues/598

**Comment by bimalkjha** [comment]
@huineng Issue is using node.js 13.x only. Please use node.js 12.x where you'll not get this issue. I am working on it to fix it. Thanks.

**Comment by huineng** [comment]
for sure you are thanks.
but just to be clear. only the new version doesn't work with node 13, the previous version does.

**Comment by mitchdennett** [comment]
I’m experiencing the same issue using NodeJs 12.13.1 

See https://github.com/ibmdb/node-ibm_db/issues/598

**Comment by huineng** [comment]
ok in that case it looks to me it's not working with nor 12.x nor 13.x, better not to upgrade then till Bimal figures it out

**Comment by bimalkjha** [comment]
@huineng @mitchdennett Please pull the latest code from git and verify. Thanks.

**Comment by mitchdennett** [comment]
@bimalkjha For me it get's past that error. But I have a new error now. 
![Screen Shot 2019-12-10 at 8 39 21 AM](https://user-images.githubusercontent.com/16268619/70534264-af2fab00-1b28-11ea-8d23-c6bdc33f56ee.png)

**Comment by huineng** [comment]
I can confirm that this pull worked

**Comment by bimalkjha** [comment]
Thanks @huineng . We have released ibm_db@2.6.3 with fix of this issue. Hence, closing it.
